tests/test_solver.py

```diff
@@ -1,641 +1,694 @@
-import claripy
-import nose
-import math
+#pylint: disable=no-self-use
+#pylint: disable=line-too-long
+#pylint: disable=missing-class-docstring
+#pylint: disable=raise-missing-from
 
+import unittest
 import logging
+import nose
+import claripy
+
 l = logging.getLogger('claripy.test.solver')
 
 solver_list = (claripy.Solver, claripy.SolverReplacement, claripy.SolverHybrid, claripy.SolverComposite, claripy.SolverCacheless)
 
-def test_solver():
-    for s in solver_list:
-        yield raw_solver, s, True
-        yield raw_solver, s, False
+class TestSolver(unittest.TestCase):
```

Comment:
The test method names in this class can be improved. They're kinda redundant and not normalized to snake case. For example, `test_solver_solver_SolverHybrid` could probably become `test_solver_hybrid`.